subprojects/integ-test/src/crossVersionTest/groovy/org/gradle/integtests/TaskSubclassingBinaryCompatibilityCrossVersionSpec.groovy

```diff
@@ -213,4 +215,45 @@ apply plugin: SomePlugin
         version previous requireGradleDistribution() withTasks 'assemble' inDirectory(file("producer")) run()
         version current requireGradleDistribution() withTasks 't' run()
     }
+
+    @Issue("https://github.com/gradle/gradle/issues/6027")
+    def "task can use project.file() from statically typed Groovy"() {
+        // Don't run these for RC 3, as stuff did change during the RCs
+        Assume.assumeFalse(previous.version == GradleVersion.version("2.14-rc-3"))
```

Comment:
I am pretty sure we are not running tests against `2.14` RCs, now that `2.14` is out.